charts/hedgedoc/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if not .Values.existingSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hedgedoc-secrets
+type: Opaque
+data: {}
```

Comment:
```suggestion
data:
  CMD_DB_PASSWORD: {{ .Values.externalDatabase.password | b64enc | quote }}
```

Reply:
if we do this, we also gotta do templating/differentiating between integrated and external db